.circleci/config.yml

```diff
@@ -213,6 +219,7 @@ commands:
         enum: ["local", "test", "staging"]
     steps:
       - install_e2e_dependencies
+      - browser-tools/install-browser-tools
```

Comment:
Some steps use hyphens and others use underscores. I'd pick one style and stick with it. Underscores seem  safer to me, and more standard as identifiers in  mosts contexts.

Reply:
Good observation. circleci seems to use hyphens for everything. We use underscores. If we want to keep all consistent, we have to change to hyphen to match. It would be in a separate PR.

Reply:
@jaycarlton ticket to track hyphen refactor. [RW-5689](https://precisionmedicineinitiative.atlassian.net/browse/RW-5689)